yt/visualization/volume_rendering/tests/test_save_render.py

```diff
@@ -0,0 +1,39 @@
+import os
+import tempfile
+import shutil
+import yt
+from yt.testing import fake_random_ds
+from unittest import TestCase
+
+def setup():
+    """Test specific setup."""
+    from yt.config import ytcfg
+    ytcfg["yt", "__withintesting"] = "True"
+
+
+class SaveRenderTest(TestCase):
+    # This toggles using a temporary directory. Turn off to examine images.
+    use_tmpdir = True
+
+    def setUp(self):
+        if self.use_tmpdir:
+            self.curdir = os.getcwd()
+            # Perform I/O in safe place instead of yt main dir
+            self.tmpdir = tempfile.mkdtemp()
+            os.chdir(self.tmpdir)
```

Comment:
I tend to see `cd`s as weak points in a test function. Could you instead save to the tmpdir directly ?

Reply:
I agree -- I wasn't entirely sure what exactly we should test here... so I copied over the test for `sigma_clip` as a starting point. So yes, can save to tmpdir directly.